### ISSUE TITLE
Fails to load ActiveSupport::Inflector

### ISSUE THREAD

**Issue by kaspernj** [mikel/mail#574]
A script I have been using to filter mails on a Courier server suddenly began to fail with the following after upgrading some gems:

```
#<NameError: uninitialized constant ActiveSupport::Inflector>
/var/lib/gems/1.9.1/gems/activesupport-4.0.0/lib/active_support/inflections.rb:4:in `<module:ActiveSupport>'
/var/lib/gems/1.9.1/gems/activesupport-4.0.0/lib/active_support/inflections.rb:3:in `<top (required)>'
/usr/lib/ruby/1.9.1/rubygems/custom_require.rb:36:in `require'
/usr/lib/ruby/1.9.1/rubygems/custom_require.rb:36:in `require'
/var/lib/gems/1.9.1/gems/activesupport-4.0.0/lib/active_support/inflector/methods.rb:4:in `<top (required)>'
/usr/lib/ruby/1.9.1/rubygems/custom_require.rb:36:in `require'
/usr/lib/ruby/1.9.1/rubygems/custom_require.rb:36:in `require'
/var/lib/gems/1.9.1/gems/activesupport-4.0.0/lib/active_support/dependencies/autoload.rb:1:in `<top (required)>'
/usr/lib/ruby/1.9.1/rubygems/custom_require.rb:36:in `require'
/usr/lib/ruby/1.9.1/rubygems/custom_require.rb:36:in `require'
/var/lib/gems/1.9.1/gems/activesupport-4.0.0/lib/active_support.rb:25:in `<top (required)>'
/usr/lib/ruby/1.9.1/rubygems/custom_require.rb:36:in `require'
/usr/lib/ruby/1.9.1/rubygems/custom_require.rb:36:in `require'
/usr/lib/ruby/1.9.1/knj/autoload/activesupport.rb:2:in `<top (required)>'
/var/lib/gems/1.9.1/gems/activesupport-4.0.0/lib/active_support/lazy_load_hooks.rb:1:in `<top (required)>'
/usr/lib/ruby/1.9.1/rubygems/custom_require.rb:36:in `require'
/usr/lib/ruby/1.9.1/rubygems/custom_require.rb:36:in `require'
/var/lib/gems/1.9.1/gems/activesupport-4.0.0/lib/active_support/i18n.rb:6:in `<top (required)>'
/usr/lib/ruby/1.9.1/rubygems/custom_require.rb:36:in `require'
/usr/lib/ruby/1.9.1/rubygems/custom_require.rb:36:in `require'
/var/lib/gems/1.9.1/gems/activesupport-4.0.0/lib/active_support/inflector/inflections.rb:3:in `<top (required)>'
/usr/lib/ruby/1.9.1/rubygems/custom_require.rb:36:in `require'
/usr/lib/ruby/1.9.1/rubygems/custom_require.rb:36:in `require'
/var/lib/gems/1.9.1/gems/activesupport-4.0.0/lib/active_support/inflector.rb:2:in `<top (required)>'
/usr/lib/ruby/1.9.1/rubygems/custom_require.rb:59:in `require'
/usr/lib/ruby/1.9.1/rubygems/custom_require.rb:59:in `rescue in require'
/usr/lib/ruby/1.9.1/rubygems/custom_require.rb:35:in `require'
/var/lib/gems/1.9.1/gems/mail-2.5.4/lib/mail.rb:37:in `<module:Mail>'
/var/lib/gems/1.9.1/gems/mail-2.5.4/lib/mail.rb:2:in `<top (required)>'
/usr/lib/ruby/1.9.1/rubygems/custom_require.rb:59:in `require'
/usr/lib/ruby/1.9.1/rubygems/custom_require.rb:59:in `rescue in require'
/usr/lib/ruby/1.9.1/rubygems/custom_require.rb:35:in `require'
/home/kaspernj/www/mailadmin/scripts/mailfilter.rb:21:in `<main>'
```

Changing the following line in `lib/mail.rb` fixes the problem:

``` ruby
  # Only load our multibyte extensions if AS is not already loaded
  if defined?(ActiveSupport)
```

To:

``` ruby
  # Only load our multibyte extensions if AS is not already loaded
  if false && defined?(ActiveSupport)
```

I suggest cutting `ActiveSupport::Inflector` away, since I can understand from various Google-searches, that this is no longer supported?

Thanks for a great library.

**Comment by yebyen** [comment]
This issue bit me from the gmail gem.  In nu7hatch/gmail#132 I surmised it was a problem with the gmail gem, but it seems to be this library instead, which gmail depends on.

**Comment by bootstraponline** [comment]
:+1:

**Comment by gerrywastaken** [comment]
@yebyen I believe your issue was fixed in Mail v2.6.4 by this change: https://github.com/mikel/mail/pull/830#discussion-diff-45785453L394

**Comment by jeremy** [comment]
Mail no longer depends on Active Support, obviating this dangling dependency issue.
